### PR TITLE
KIALI-319 Change swsui code, docs, scripts, etc. to reflect the new name

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,9 +21,9 @@ deploy:
     secure: X0tQXL0stFL2cE+PHM6ZoqDHSgHQEQPJ/uD5FZJanueZ/w9Q2QxVYg9fYj3ITBKZX6DEd95Qsnfx8rwnMf8LJG+snDPhv0oCUWuZAgtaz3W59teAA195nNTkWWrEnIiR1eWZo/apxImSHfdJmjdvhcMYHDAidalNVMOwcTNgDvwmAC55S3CI9mhVqE/7zTJHTWyz4p+tWYlHJTzMTNQMfKrcbYzT9pyWUpKcgRtFDOBdq5Vp/AJahT3NajU4G+UoVOIzq8iDB+q81fnIF8TbV/RrUFB5fLHkYGBo4bw0UOoM2kH7og9Xe6T0GOgBT84bCYQ+V+xkCJIPs/d8/w1dm5fBtzJKsHjFHMDEan2oDNEAN1B1o7sZ0mpVpbBc1/caYLYXJ6GO8L2VpnXFHBOPzOEWi7HPLL5ydn0KQHUR9YeX2jyFZmlhXd7+xlRSHqZIPC0zGQ8JStvz9z5lx02/Oq/1fmRLPf1i8dWmAJkIgTzFa2FRCVhzQZ/uHD64paF81mcOeVq6y/swcoYvj+G+CNyBnAMn1Cgu6eelsHtnr3TzwROHCLz3pFhp2IL32O+Awt/f8F3OaOa+SNzWtHn6XW8rtsVrgWj8yki6AqntRs3fuOBTyUReJRLuyPjCfbVjGNKNljX7lf53WUWD/eAlQa6pAOjtz97DGi1VkTAQer4=
```
Comment (jmazzitelli) [inline .travis.yml]: @josejulio now we have a new repo (kiali/kiali-ui) you will need to run the "travis encrypt" tool to get a new key.
We also need @kiali/kiali-ui project created in npm repo.

Comment (josejulio) [reply .travis.yml]: Already did that and tested locally :)

Comment (josejulio) [reply .travis.yml]: Just to clarify, the same key should work for the new org, is my key.

**Comment by jmazzitelli** [comment]
This PR blocks https://github.com/kiali/kiali/pull/132

Once we merge this PR and we see the new kiali-ui published on npm, that PR https://github.com/kiali/kiali/pull/132 can then be merged.

File: .travis.yml (hunk 1 above)
Comment (jmazzitelli) [reply .travis.yml]: OK, I'll merge and we'll see.

**Pull request merged**